bin/generate-feature-config.php

```diff
@@ -0,0 +1,28 @@
+<?php
+/**
+ * Generates an array of feature flags, based on the config used by the client application.
+ *
+ * @package WooCommerce Admin
+ */
+
+$phase = isset( $_SERVER['WC_ADMIN_PHASE'] ) ? $_SERVER['WC_ADMIN_PHASE'] : ''; // WPCS: sanitization ok.
+if ( empty( $phase ) || ! in_array( $phase, array( 'development', 'plugin', 'core' ) ) ) {
```

Comment:
Good catch -- the in_array is enough here. I get in the habit of checking variables in PHP to avoid throwing notices, but in this case I just defined it so it's unnecessary.

Fixed in https://github.com/woocommerce/wc-admin/pull/1500/commits/45badff93d44a9920192796be5875bab131c0a93.